lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -129,7 +129,6 @@ export default function FilePickerApp({
           authToken={authToken}
           authUrl={authUrl}
           courseId={courseId}
-          lmsUrl={lmsUrl}
```

Comment:
~Was this just un-used?~ No`customCanvasApiDomain` and `lmsUrl` have been merged into `origin`